conjur/cli.py

```diff
@@ -28,14 +32,16 @@ class Cli():
     Main wrapper around CLI-like usages of this module. Provides various
     helpers around parsing of parameters and running client commands.
     """
+    LOGGING_FORMAT = '%(asctime)s %(levelname)s: %(message)s'
 
     @staticmethod
-    def main_description():
+    def usage(args):
```

Comment:
`args` is a controversial name for a parameter if you don't use it in the common way like `def a(*args, **kwargs)`. Maybe change it to `command_args`?

Reply:
sure! Updating now

Reply:
I mean ideally I could use *args because it is not keyworded